Userland/Applications/Escalator/EscalatorWindow.cpp

```diff
@@ -57,7 +57,10 @@ EscalatorWindow::EscalatorWindow(StringView executable, Vector<StringView> argum
     size_t password_attempts = 0;
     m_ok_button->on_click = [this, password_attempts](auto) mutable {
         if (check_password()) {
-            MUST(execute_command());
+            auto result = execute_command();
+            if (result.is_error()) {
+                GUI::MessageBox::show_error(this, ByteString::formatted("Failed to execute command: {}", result.error()));
+            }
```

Comment:
This should be squashed into the previous commit. Per CONTRIBUTING.md:
> Amend your existing commits when adding changes after a review

Reply:
Done. Sorry I was in work mode where we do new commits :)